projects/core/src/model/cms.model.ts

```diff
@@ -54,6 +54,16 @@ export interface CmsSiteContextSelectorComponent extends CmsComponent {
   context?: string;
 }
 
+export enum ScrollBehavior {
+  AUTO = 'auto',
+  SMOOTH = 'smooth',
+}
+
+export interface CmsScrollToTopComponent extends CmsComponent {
+  behavior?: ScrollBehavior;
```

Comment:
perhaps name the variable `scrollBehavior` to make it self explanatory.

Reply:
Yes, I will rename it